workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -64,12 +64,20 @@ public void Awake()
                 var workerId =
                     CommandLineUtility.GetCommandLineValue(commandLineArgs, RuntimeConfigNames.WorkerId,
                         string.Empty);
+                var useUniqueConnectionId = CommandLineUtility.GetCommandLineValue(commandLineArgs,
+                    RuntimeConfigNames.UseUniqueConnectionId,
+                    "false") == "true";
 
                 // because the launcher does not pass in the worker type as an argument
                 var worker = workerType.Equals(string.Empty)
                     ? WorkerRegistry.CreateWorker<UnityClient>($"{workerType}-{Guid.NewGuid()}", new Vector3(0, 0, 0))
                     : WorkerRegistry.CreateWorker(workerType, workerId, new Vector3(0, 0, 0));
 
+                if (useUniqueConnectionId)
+                {
+                    worker.RequiresUniqueWorkerIdPerConnection = true;
```

Comment:
I don't think this is the best variable name, all connections require a unique id because if you try to use the same one then you can't reconnect so it might mislead people. Maybe something like: `AllowDynamicId`

Reply:
Aye, I like the suggestion

Reply:
Also moving the declaration to top of class